spec/features/admin/councils/new_spec.rb

```diff
@@ -66,22 +66,23 @@
       # Only Email filled in
       it do
         fill_in 'Email', with: 'C1@council.co.nz'
-        click_button 'Save'
+        expect { click_button 'Save' }.to change { Council.count }.by(0)
```

Comment:
```suggestion
         expect { click_button 'Save' }.not_to change { Council.count }
```

Reply:
Rubocop throws an error when that is implemented :frowning_face: